test/iron-ajax.html

```diff
@@ -336,6 +336,100 @@
           expect(server.requests[0]).to.be.ok;
           expect(server.requests[0].requestBody).to.be.equal(requestBody);
         });
+
+        suite('and `body` is a plain Object', function() {
+          test('if `contentType` isn\'t set, the body is form encoded',
+            function() {
```

Comment:
I'm not personally opposed to this formatting style, but we do not use it in any of our other test suites. We should make this case stylistically consistent.